pkg/users/accounts-list.js

```diff
@@ -82,7 +82,7 @@ const UserActions = ({ account }) => {
     );
 
     const kebab = (
-        <Dropdown toggle={<KebabToggle onToggle={setKebabOpen} />}
+        <Dropdown toggle={<KebabToggle onToggle={(_, isOpen) => setKebabOpen(isOpen)} />}
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18721-20230426-094458-1180deae-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)